Highlight Canvas features (#8799)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/use-canvas.md

1. Drag an [operator](/docs/cloud/canvas-interface#operators) from the operator toolbar and drop it onto the canvas.
2. Click on the operator to open its configuration panel.

> 3. View the **Output** and **Code** tabs to validate your transformation:

Reviewer: I don't think these are "Tabs." We seem to refer to them as "side panels" elsewhere but we might want to just call them panels or refer to this as "edit mode." I see a "close edit mode" button when I'm in edit mode.

	- The **Output** tab shows row counts, column details, and a data preview for each operator. Use this to verify your transformation produces the expected results.
	- The **Code** tab displays the SQL code generated by the node's configuration.
4. Connect the operators by using the connector by dragging your cursor between the operator's "+" start point and linking it to the other operators you want to connect to. This should create a connector line. 
	- Doing this allows the data to flow from the source table through various transformations you configured, to the final output.
5. Keep building your dbt model and validate each step using the **Output** tab. Check the row count at each operator to catch unexpected data changes early.

## Validate transformations step-by-step

<Constant name="canvas" /> lets you inspect your data at every transformation step, making it easy to build confidence in your models before deploying them:

> - **Check row counts:** After adding each operator, click on it and view the **Output** tab to see the row count. This immediately reveals:

Reviewer: ```suggestion
- **Check row counts:** Click an operator after adding it and view the **Output** tab to see the row count. This immediately reveals:
```

> - **Check row counts:** After adding each operator, click on it and view the **Output** tab to see the row count. This immediately reveals:

Reviewer: @matthewshaver this does not seem to be a "tab." Using tab makes it confusing.
